backend/src/test/java/eu/bbmri_eric/negotiator/attachment/AttachmentConversionServiceTest.java

```diff
@@ -991,4 +991,622 @@ private byte[] createRealisticDocBytes() {
 
     return docContent;
   }
+
+  // ============================================
+  // XLSX Converter Tests
+  // ============================================
```

Comment:
Feel free to create a new test file, instead of separation with comments, this will get outdated really quickly